### ISSUE TITLE
ical needs updated to 7.0.3

### ISSUE THREAD

**Issue by barrelltitor** [humbertogontijo/homeassistant-roborock#616]
## Version of the custom_component
1.0.13

## What vacuum are you using?
Irrelevant

## Describe the bug

ical needs updated to 7.0.3. On linuxserver's docker container at least, this installs the older version and breaks the functionality of the local_todo integration

## Debug log

irrelevant

**Comment by AlessandroTischer** [comment]
> ## Version of the custom_component
> 1.0.13
> 
> ## What vacuum are you using?
> Irrelevant
> 
> ## Describe the bug
> 
> ical needs updated to 7.0.3. On linuxserver's docker container at least, this installs the older version and breaks the functionality of the local_todo integration
> 
> ## Debug log
> 
> irrelevant

Same issue here

**Comment by barrelltitor** [comment]
To fix it for now just manually install the version, off the top I think it was 
'pip install ical==7.0.3'
Or change the ical version in the file of the custom component then restart the container for a persistent fix

**Comment by AlessandroTischer** [comment]
> Or change the ical version in the file of the custom component then restart the container for a persistent fix

Inside the manifest file? Thank you by the way!

**Comment by barrelltitor** [comment]
Yeah in custom_components/roborock/manifest.json, then restart your container. If you're using lscr.io's it should auto install the correct dependency

**Comment by Lash-L** [comment]
This is fine but in reality iCal should be ripped out as it isn't used. And just updating the version is just pushing the problem down the road. We don't actually use iCal it was for a entity we almost added but didn't (calendar) so the calendar stuff in general just needs to be removed

**Comment by ScottG489** [comment]
I was surprised to see this fixed, but there hasn't been a release since November. Just curious, but is there a reason a release hasn't been cut in so long? I wasn't able to see a mention of anything regarding why.

Is it that the the core integration has caught up with the features here?

**Comment by Lash-L** [comment]
I have limited time so all of my time is spent on the core integration - and i don't think humberto has much time either - so no time is spent on the custom integration

**Comment by fxfitz** [comment]
@Lash-L @humbertogontijo I'm also being bit by this. If you and humberto don't have any time to work on a release for this, can I help in some way? Would really like to see this fix get pushed out. :-)
